tests/flow/test_stress.py

```diff
@@ -23,8 +23,9 @@ def query_crud(graph, threadID):
             graph.query(read_query)
             graph.query(update_query)
             graph.query(delete_query)
+            return True
```

Comment:
`def query_crud(graph, threadID):`
please rename` threadID` to something more relevant/meaningful